prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -33,11 +32,17 @@ import gov.cdc.prime.router.transport.RetryItems
 import gov.cdc.prime.router.transport.RetryToken
 import gov.cdc.prime.router.transport.SftpTransport
 import gov.cdc.prime.router.transport.SoapTransport
+import org.apache.logging.log4j.kotlin.Logging
 import org.jooq.Configuration
 import org.jooq.Field
 import java.io.ByteArrayInputStream
 import java.time.OffsetDateTime
 
+/**
+ * Keyword used to trigger alarms when an Exception cannot easily be thrown.
+ */
+const val ALERT_TRIGGER = "AlertTheHumans"
+
```

Comment:
This is the super secret magic word.  Could move this to an env var if you wish.

Reply:
IMO having it static is fine. Pretty sure this won't occur naturally elsewhere :-)

Reply:
Sounds good to me, another option is to use logging levels. Does the logger interface have a `fatal` or something?

If we stick with the magic work is there a good place to put it in the codebase so everything that want's to use it doesn't have to depend on WorkflowEngine?

Reply:
Its outside of the WorkflowEngine class, so I think it should be available.

Reply:
Also, I think it belongs in .../azure/ since this is really only needed in the cloud.

Reply:
I don't think using `fatal` is a great option because there are lots of fatal exceptions that are already triggering PagerDuty with no problems.     I thought about adding a new level, but this feature didn't seem worth that level of effort.